pages/contribute/index.html

```diff
@@ -4,5 +4,24 @@
 {{define "body"}}
 <article>
 <h1>How to Contribute</h1>
+
+<p>Team Silverblue is an open community seeking contributors from everywhere, and of every type.
+In addition to many core projects needing code in various languages, we also need:
+<ul>
+  <li>help writing documentation</li>
+  <li>expert users to answer questions from less experienced users</li>
+  <li>testers who can try out new releases and features</li>
+</ul>
+</p>
+
+<p>If you are a programmer, please check out any of the many projects under the
+Project Atomic or Flathub organizations on GitHub for places to contribute.
```

Comment:
IMHO, worth adding some links here to the said github orgs (reduce friction, better google rank, etc)